python/docs/source/reference/pyspark.pandas/general_functions.rst

```diff
@@ -52,18 +53,25 @@ Top-level missing data
 .. autosummary::
    :toctree: api/
 
-   to_numeric
    isna
    isnull
    notna
    notnull
 
-Top-level dealing with datetimelike
+Top-level dealing with numeric data
 -----------------------------------
+
+.. autosummary::
+   :toctree: api/
+
+   to_numeric
+
+Top-level dealing with datetimelike data
+----------------------------------------
```

Comment:
Just for matching pandas documents as below:

<img width="618" alt="Screen Shot 2022-04-01 at 1 22 51 PM" src="https://user-images.githubusercontent.com/44108233/161194528-322aa739-12a1-4fb3-9741-05b2124f38bc.png">